WordPress/Classes/ViewRelated/Post/PostEditor+MoreOptions.swift

```diff
@@ -31,6 +31,42 @@ extension PostEditor {
     }
 
     private func savePostBeforePreview(completion: @escaping ((String?, Error?) -> Void)) {
+        guard RemoteFeatureFlag.syncPublishing.enabled() else {
+            return _savePostBeforePreview(completion: completion)
+        }
+
+        guard !post.changes.isEmpty else {
+            completion(nil, nil)
+            return
+        }
+
+        Task { @MainActor in
+            let coordinator = PostCoordinator.shared
+            do {
+                if post.isStatus(in: [.draft, .pending]) {
+                    SVProgressHUD.setDefaultMaskType(.clear)
+                    SVProgressHUD.show(withStatus: Strings.savingDraft)
```

Comment:
I slightly changed the copy to communicate to the user that we're saving the changes.

Reply:
Nice touch - makes it clear